src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx

```diff
@@ -0,0 +1,92 @@
+* Maximize data ingest
+* Keep data organized 
+* Monitor multiple environments, like staging and development
+* Manage billing across different areas of your organization
+
+This doc will guide you through setting up your account structure. Creating accounts is a good thing to do before you begin ingesting data. It can be harder to make changes after you connect New Relic to your applications because you risk losing historical context.
+
+<Callout variant="tip">
+If you're using Free or Standard editions, you only have one account available and you won't need this doc but feel free to revisit if you decide to change to [Pro or Enterprise](https://newrelic.com/pricing).
+</Callout>
+
+## Before you get started [#begin]
+
+In this guide we're going to help you create and manage your New Relic account. Before you get started we want to make sure you're familiar with our terms:
+
+* **Organization**:  an organization represents a New Relic customer. Your New Relic organization contains everything relevant to you and your team members: your organization's accounts, users, and data.
+* **Account**: When you sign up for New Relic your organization is automatically assigned one account. An account can have many sources of data reporting to it. For example, a single account could have data reporting from our infrastructure agent, an APM agent, and other integrations. All data reported to New Relic requires an account ID, which tells New Relic which account that data belongs in. That ID is also used for some account-specific tasks, like making API calls.
+* **User**: In your organization, your New Relic users are granted access to specific accounts that are relevant to their duties and responsibilities. To manage users' access to accounts, you grant groups access to specific roles on specific accounts. Our user management system allows you to create the user access you need, whether that's a relatively simple setup with just a few roles across a few accounts, or a complex one with many roles across many accounts. Learn more about user management.
+
+## What account structure is right for me? [#account-structure]
+
+Signing up for New Relic automatically generates an organization. An organization contains one account unless you manually create more. One account can be perfect for many companies and it might not be necessary for you to add more. 
+
+Some indications that one account is right for you:
+
+* You have 200 employees or less and you don't anticipate growing too much
+* You aren't ingesting vast amounts of data. If you're not sure, use [our spreadsheet](https://newrelic.com/blog/nerdlog/estimate-data-cost) to estimate the amount of data you might send to New Relic
+* You expect to send a moderate amount of alerts. If you're not sure, review our [alerting rules and limits](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts/)
+
+If these don't apply to you, it might be good to consider a multiple account structure. 
```

Comment:
I might rephrase this to not be negative. 'If these apply to you, then you only need one account. Otherwise, we recommend considering a multi-account structure to fit your needs"

Reply:
also is it one or all need to apply? If yoou have 200 employees or less but you're still consuming a lot of data, would it be better to be multiaccount? Just something to consider

Reply:
What if I say: If these apply to you, then you `might` only need one account. Otherwise, we recommend considering a multi-account structure to fit your needs.

Reply:
Yes, I think that's a good halfway point. Esp since you can have a separate section about multiple accounts